# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/chrome-webidl/WindowGlobalActors.webidl
# Commit: a902acfbc868
# Full Hash: a902acfbc8680811e4538f8d27e78e4fe6ca55f2
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 15:55:11
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 13 - Stop inheriting nsIRemoteTab interface in BrowserParent. r=nika
#   
#   This commit removes nsIRemoteTab as a parent class from BrowserParent,
#   so that BrowserHost is the only concrete implementation of nsIRemoteTab.
#   
# ==============================================================================

diff -r 50475885250c -r a902acfbc868 dom/chrome-webidl/WindowGlobalActors.webidl
--- a/dom/chrome-webidl/WindowGlobalActors.webidl	Wed May 08 11:45:23 2019 -0500
+++ b/dom/chrome-webidl/WindowGlobalActors.webidl	Wed May 08 14:34:47 2019 -0500
@@ -23,8 +23,6 @@
 
   readonly attribute WindowGlobalChild? childActor; // in-process only
 
-  readonly attribute RemoteTab? remoteTab; // out-of-process only
-
   // Information about the currently loaded document.
   readonly attribute Principal documentPrincipal;
   readonly attribute URI? documentURI;